plugins/node/opentelemetry-instrumentation-mysql/test/index.test.ts

```diff
@@ -152,6 +152,66 @@ describe('mysql@2.x', () => {
         });
       });
     });
+
+    it('db.statement should have ? instead of quoted words when using default serializer ', done => {
+      const span = provider.getTracer('default').startSpan('test span');
+      context.with(trace.setSpan(context.active(), span), () => {
+        const sql = 'SELECT * FROM `books` WHERE `author` = ?';
+        const query = connection.query({ sql, values: ['David'] });
+        query.on('end', () => {
+          const spans = memoryExporter.getFinishedSpans();
+          assert.strictEqual(spans[0].name, sql);
+          assert.strictEqual(
+            spans[0].attributes[SemanticAttributes.DB_STATEMENT],
+            'SELECT * FROM `books` WHERE `author` = ?'
+          );
+          done();
+        });
+      });
+    });
```

Comment:
Maybe move this test below to the `describe('dbStatementSerializer config'` part? I think it's more appropriate there as it tests the default config serializer